### ISSUE TITLE
Book 4 should be Hermione Jean Granger and the Phoenix's call

### ISSUE THREAD

**Issue by jrincayc** [ianstormtaylor/hpmor#5]
The cover of book 4 should say Hermione Jean Granger and the Phoenix's Call, not Harry Potter and the Phoenix's Call.  The Cover should match the title page.

**Comment by jrincayc** [comment]
A second change possibly should be that book 2 should be: "Harry James Potter-Evans-Verres and the Professors' Games" instead of "Harry James Potter-Evans-Verres and the Professor's Games"

There are more than one professor playing games in book 2.

**Comment by ianstormtaylor** [comment]
Hey @jrincayc, I'm sorry I only just saw the notice about book four actually being named "Hermione Granger ...". I've just updated it! Thank you!

(I chose to leave book two the way it was, since it seems that's how the original source was named. I also chose to continue leaving out the longer middle/last names on the cover design for aesthetics, although I hadn't even realized that I omitted that originally.)
